src/main/java/org/spongepowered/api/CatalogType.java

```diff
@@ -60,5 +64,13 @@
      * @return The human-readable name of this catalog type
      */
     String getName();
+    
+    @Override
+    default Text toText() {
```

Comment:
If the developer knows what they are including in a `Text` object, then why not leave it up to the developer to decide on how they want a `CatalogType` to be seen as `Text`. `CatalogType` is so ultimately vague that no one can dictate how it should be seen as a `Text` object short of writing `Text` as if it's the `toString()` including all the known variables of the `CatalogType`, which honestly is much more information than needed to present itself as a `Text` object.